build.sbt

```diff
@@ -6,7 +6,7 @@ description := "sbt plugin for cassandra schema/data migrations using pillar (ht
 
 organization := "io.ino"
 
-version := "2.0.0"
+version := "2.1.0"
```

Comment:
Given that the project did not have a pr since May I assumed that there are no changes in the queue and given that I required this anyway for local testing via `publishLocal` I left it in there and mentioned it explicitly in the comment for the pr, but never mind ;)